src/components/AddButton.js

```diff
@@ -0,0 +1,53 @@
+import React,  { Component, } from "react";
+import PropTypes from "prop-types";
+import { Button, View, TouchableOpacity, Text, Image} from 'react-native';
+import styled from 'styled-components';
+import AddIcon from '../assets/images/icons/add-icon.png';
+
+class AddButton extends Component {
+    constructor(props) {
+      super(props);
+    }
+  
+    render() {
+      return (
+      <AddCustomButton type="submit" >
+          <Image style={{ backgroundColor: 'transparent', tintColor: '#5089E9', bottom: 10, height: 40, width: 40 }} source= {AddIcon}/>
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Airbenders-490/AirbendersFE/pullRequest?prid=8247201)
```suggestion
          <Image style={{ backgroundColor: "transparent", tintColor: '#5089E9', bottom: 10, height: 40, width: 40 }} source= {AddIcon}/>
```